docs/integrations/aiohttp.md

```diff
@@ -39,6 +39,24 @@ if __name__ == "__main__":
 
 The keyword arguments of `logfire.instrument_aiohttp_client()` are passed to the `AioHttpClientInstrumentor().instrument()` method of the OpenTelemetry aiohttp client Instrumentation package, read more about it [here][opentelemetry-aiohttp].
 
+## Masking URL in AIOHTTP span
+To mask a URL for sensitive data you add an AIOHTTP OpenTelemetry URL filter.
+
+```python
+from yarl import URL
+
+
+def mask_url(url: URL) -> str:
+    sensitive_keys = {"username", "password", "token", "api_key", "api_secret", "apikey"}
+    masked_query = {key: "*****" if key in sensitive_keys else value for key, value in url.query.items()}
+    return str(url.with_query(masked_query))
+```
+
+Now configure logfire to use this URL filter
+```python
+logfire.instrument_aiohttp_client(url_filter=mask_url)
+```
```

Comment:
```suggestion
## Hiding sensitive URL parameters

The `url_filter` keyword argument can be used to modify the URL that's recorded in spans. Here's an example of how to use this to redact query parameters:

```python
from yarl import URL

def mask_url(url: URL) -> str:
    sensitive_keys = {"username", "password", "token", "api_key", "api_secret", "apikey"}
    masked_query = {key: "*****" if key in sensitive_keys else value for key, value in url.query.items()}
    return str(url.with_query(masked_query))

logfire.instrument_aiohttp_client(url_filter=mask_url)
```